guides/covariates_tfms.md

```diff
@@ -0,0 +1,289 @@
+                                  output_chunk_length=1, 
+                                  **model_kwargs)
+```
+
+All TFMs can be trained on single or multiple `target` series and, depending on their covariate support, `past_covariates` and / or `future_covariates`.
+
+```
+# fit the model on a single target series with optional past and / or future covariates
+model.fit(target, 
+          past_covariates=past_covariates, 
+          future_covariates=future_covariates)
+```
+
+You can produce forecasts for any input `target` TimeSeries (also unobserved targets) as long as it contains `input_chunk_length` time steps.
+
+```
+# predict the next n=3 time steps for any input series with `series`
+prediction = model.predict(n=3, 
+                           series=target, 
+                           past_covariates=past_covariates, 
+                           future_covariates=future_covariates)
+```
+
+If you want to know more about the training and prediction process of our Torch Forecasting Models and how they work with covariates, read on.
+
+## Content of this document
+
+[Section 1](#1-torch-forecasting-models) covers the most important points about Torch Forecasting Models:
+- how they work on a top-level
+- Torch Forecasting Model covariates support
+- time span requirements for target and covariate series
+
+[Section 2](#2-in-depth-look-at-how-past-and-future-covariates-are-used-in-a-torch-forecasting-model) gives
+an in-depth guide of how covariates are used in Darts' PyTorch-based Forecasting Models.
+
+## 1. Torch Forecasting Models
+## 1.1. Introduction
+TFMs are broadly speaking "machine learning based" models, which denote PyTorch-based (deep learning) models.
+
+TFMs train and predict on fixed-length chunks (sub-samples) of the `target` and `*_covariates` series (if supported). `Target` is the series for which we want to predict the future, `*_covariates` are the past and / or future covariates (explanatory variables) that can help models improve forecasts. 
+
+Each chunk contains an input chunk - resembling the sample's past - and an output chunk - the sample's future. The sample's prediction point lies at the end of the input chunk. The length of these chunks has to be specified at model creation with parameters `input_chunk_length` and `output_chunk_length`.
+
+```
+# model with chunks looking 7 time steps back (past) and 1 time step ahead (future)
+model = SomeTorchForecastingModel(input_chunk_length=7, output_chunk_length=1)
+```
+
+Depending on their capabilities, TFMs accept past and / or future covariates as optional parameters `past_covariates` and `future_covariates` in `fit()` and `predict()`. You can find the covariate support for each TFM in [section 1.2.](#12-torch-forecasting-model-covariates-support). 
+
+***The same covariates can be used for training and prediction if they have the required time spans (explained in [section 1.3.](#13-required-target-time-spans-for-training-validation-and-prediction))***
```

Comment:
Here too this is somewhat of a repetition from the higher-level document, which I would assume they would have read first (as you link to it at the top, saying it's assumed the read it).